src/content/protocols/sky.md

```diff
@@ -0,0 +1,387 @@
+| Clipper                               | File                   | Changes contract parameters or stops the contract. This function can be used to update the address of auxialary contracts or auction parameters that regard the timing, speed, fees, and incentives of auctions. The contract can also be stopped gradually to first stop new auctions, then auction remakes, and then bids.                                                                                                                           | DSPauseProxy (DAO), Dog, ClipperMom, ESM, End                                               |
+| Clipper                               | Kick                   | Starts a new auction. This function is meant to be called by `Dog`. There are no particular input validations. If this contract is directly called by the governance (bypassing `Dog`), it could liquidate a healthy debt position.                                                                                                                                                                                                                    | DSPauseProxy (DAO), Dog, ClipperMom, ESM, End                                               |
+| Clipper                               | Yank                   | Cancels an auction. All auctions would be cancelled in case of an Emergency Shutdow (through `End`).                                                                                                                                                                                                                                                                                                                                                   | DSPauseProxy (DAO), Dog, ClipperMom, ESM, End                                               |
+
+## Dependencies
+
+The Sky protocol relies on the provider Chronicle for price feeds of collateral assets. An Oracle Security Module enforces a 1 hour window on price updates and the provider (Chronicle) or the governance can freeze the current price value to prevent further updates.
+
+Chronicle is a decentralized oracle protocol that computes a median price from multiple sources. The protocol contains validators who push new prices and challengers who can freeze and challenge new prices. The contracts are non-upgradeable. In addition to freezing prices, the MakerDAO governance can change the oracle provider with a governance proposal.
+
+## Exit Window
+
```

Comment:
structure this similar to how explained in the overview section. be more detailed here and only summarize in the overview section